docs/source/1.0/spec/core/idl.rst

```diff
@@ -1438,6 +1445,97 @@ For example:
     string SensitiveText with [SensitiveString]
 
 
+.. _idl-target-elision:
+
+Target Elision
```

Comment:
A structure referencing a resource isn't useful by itself though, so I don't think it can reasonably have its own section. And starting with the mixin case here seems more natural to me since it follows the mixin section.